core/src/main/kotlin/com/alecstrong/sql/psi/core/SqlFileBase.kt

```diff
@@ -13,9 +15,9 @@ import com.intellij.lang.Language
 import com.intellij.openapi.roots.ProjectRootManager
 import com.intellij.psi.FileViewProvider
 import com.intellij.psi.PsiElement
-import com.intellij.psi.PsiFile
 import com.intellij.psi.PsiManager
 import com.intellij.psi.util.PsiTreeUtil
+import com.intellij.util.containers.MultiMap
```

Comment:
I'm surprised IntelliJ doesn't have an IntMap or something to avoid the extra dep

Reply:
it has some things named intmap but dont look like i can iterate them in order